### ISSUE TITLE
Execution Hangs Without Seeding ETH

### ISSUE THREAD

**Issue by CamArmstr** [stevenpslade/maas#2]
When I go to create a new multisig, but I don't put in any ETH to seed it the execution hangs (spinning create wheel of death). Just a heads up.

![image](https://user-images.githubusercontent.com/52431864/163629409-ef8a0262-f02b-4073-8ae2-feb4709e5c87.png)

Thanks for putting the work into this! It's a great example to learn from. I really appreciate it.

**Comment by stevenpslade** [comment]
Good catch. I noticed a similar result when you add an empty address field. No spinning wheel of death but does not execute or show any errors.

This create modal definitely needs better error handling and UI!
